File: cryptocurrencyWeb/src/main/java/com/outsourcebooster/cryptocurrency/web/model/Wallet.java
package com.outsourcebooster.cryptocurrency.web.model;

import java.util.List;

/**
 * Created by rklimemnko on 30.05.2016.
 */
public class Wallet {
    private List<Account> accounts;

    public Wallet() {}

    public Wallet(List<Account> accounts) {
        this.accounts = accounts;
    }

    public List<Account> getAccounts() {
        return accounts;
    }

    public void setAccounts(List<Account> accounts) {
        this.accounts = accounts;
    }
}

File: cryptocurrencyWeb/src/main/java/com/outsourcebooster/cryptocurrency/web/model/user/Wallet.kt
package com.outsourcebooster.cryptocurrency.web.model.user

import com.outsourcebooster.cryptocurrency.web.model.account.CryptocurrencyAccount
import com.outsourcebooster.cryptocurrency.web.model.account.FiatAccount

/**
 * Created by rklimemnko on 30.05.2016.
 */
data class Wallet(val cryptocurrencyAccounts: List<CryptocurrencyAccount>,
                  val fiatAccounts: List<FiatAccount>) {
    constructor() : this(listOf(), listOf())
}